Since I cannot view the details or the video of this test failing, I spun up a local dev environment to try to replicate and see what's going on. Here are my findings (I'll post them in separate comments):

#### With no value set to the filter
1. Going to the dashboard from `/collection/root` results in data failing to load (it loads on mouse hover)
https://share.getcloudapp.com/NQu1PbvN (GIF screenshot)

There's all kinds of console warnings and errors
```
Uncaught Invariant Violation: findDOMNode was called on an unmounted component.
    at invariant (http://localhost:8080/app/dist/vendor.hot.bundle.js?5b5e02b9818d518b76f2:84935:15)
    at Object.findDOMNode (http://localhost:8080/app/dist/vendor.hot.bundle.js?5b5e02b9818d518b76f2:194157:21)
    at _class._getElement (http://localhost:8080/app/dist/app-main.hot.bundle.js?5b5e02b9818d518b76f2:46931:44)
    at _class.<anonymous> (http://localhost:8080/app/dist/vendor.hot.bundle.js?5b5e02b9818d518b76f2:198578:37)
    at _this._updateSize (http://localhost:8080/app/dist/app-main.hot.bundle.js?5b5e02b9818d518b76f2:46909:31)
```

Cannot read property of rows/slice/cols of undefined. (The same warning for each - don't want to repeat 3 logs)
```
settings.js:160 Error getting setting pie._metricIndex TypeError: Cannot read property 'cols' of undefined
    at Object.getValue (PieChart.jsx:157)
    at getComputedSetting (settings.js:141)
    at getComputedSettings (settings.js:99)
    at getComputedSettingsForSeries (visualization.js:89)
    at Visualization.transform (Visualization.jsx:233)
    at Visualization.componentWillMount (Visualization.jsx:166)
    at ReactCompositeComponent.js:345
    at measureLifeCyclePerf (ReactCompositeComponent.js:73)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js:344)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js:255)
```

--- a/frontend/test/metabase/scenarios/question/nulls.cy.spec.js
+++ b/frontend/test/metabase/scenarios/question/nulls.cy.spec.js
@@ -37,7 +37,11 @@ describe("scenarios > question > null", () => {
     });
   });
 
-  it("pie chart should handle `0`/`null` values (metabase#13626)", () => {
+  // [quarantine]
+  //  - possible app corruption and new issue with rendering discovered
+  //  - see: https://github.com/metabase/metabase/pull/13721#issuecomment-724931075
+  //  - test was intermittently failing
+  it.skip("pie chart should handle `0`/`null` values (metabase#13626)", () => {
     // Preparation for the test: "Arrange and Act phase" - see repro steps in #13626
     withSampleDataset(({ ORDERS }) => {
       // 1. create a question

